Mykola Rossomakha/HW7/Task 7.3.py

```diff
@@ -0,0 +1,17 @@
+def calculate_value(string):
+    """
+    Функція повертає кількість літер у рядку у формі словника.
```

Comment:
good job
look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result